scripts/copy-res.js

```diff
@@ -195,21 +163,28 @@ function genLangFile(lang, dest) {
 
     translations = weblateToCounterpart(translations);
 
-    fs.writeFileSync(dest + lang + '.json', JSON.stringify(translations, null, 4));
+    const json = JSON.stringify(translations, null, 4);
+    const jsonBuffer = Buffer.from(json);
+    const digest = loaderUtils.getHashDigest(jsonBuffer, "sha512", "base64", 7);
```

Comment:
I think `base64` includes `/`, which could get weird for things in file paths. Also, I'm not sure how much it actually impacts build time, but `sha512` seems like overkill for this problem... We really only need the simplest thing that gives a different value when the file changes. (Also we're only keeping 7 bytes of it, so powerful hashes like `sha512` are mostly doing work that is thrown out.)

For images via Webpack we're currently using `[hash:7]`, which [seems to equate to](https://github.com/webpack/loader-utils/blob/b91a76c0c98bf71df5a3a1c808ea86268c3716c1/lib/interpolateName.js#L96) `getHashDigest(buffer, null, null, 7)`, which will then default to `md5` and `hex`, which seems fast, safe, and sufficient for the problem here.